resource-timing/resources/all_resource_types.htm

```diff
@@ -6,6 +6,35 @@
         <link rel="shortcut icon" href="empty.py?favicon">
     </head>
     <body background='blue.png?id=body'>
+        <script>
+            let count = 0;
+            let totalEventSourceCount = 0;
+
+            function perf_observer(list, observer) {
+                const entries = list.getEntries();
+                const eventSourceCount = entries.filter(e => e.name.includes('eventsource')).length;
+
+                if(eventSource > 0) {
+                    totalEventSourceCount += eventSourceCount;
+                } 
+                
+                // exclude eventSource since it arrives repeatedly
```

Comment:
Oh that's weird, what do you mean by this? The same resource arrives multiple times?

Reply:
the way it is implemented it arrives multiple times so each time it calls the observer callback